### PR TITLE
CloudWatch/Athena - valid metrics and dimensions.

### PR DESCRIPTION
**Pull request by kwarunek**
**What this PR does / why we need it**:

Separate Athena's  dimensions from the metrics. 

**Which issue(s) this PR fixes**:

Grafana does not prompt with a dimension for CloudWatch/Athena

    t=2020-10-21T14:46:56+0200 lvl=eror msg="Metric request error" logger=context userId=534 orgId=275 uname=test error="unable to find dimension \"AWS/Athena\"" remote_addr=10.20.30.40

It turns out that dimensions are, incorrectly, combined with metrics. 

Change in accordance with https://docs.aws.amazon.com/athena/latest/ug/query-metrics-viewing.html.

### CODE REVIEW

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ var metricsMap = map[string][]string{
...
-	"AWS/Athena":                  {"DataScannedInBytes", "EngineExecutionTime", "QueryPlanningTime", "QueryQueueTime", "QueryState", "QueryType", "ServiceProcessingTime", "TotalExecutionTime", "WorkGroup"},
+	"AWS/Athena":                  {"DataScannedInBytes", "EngineExecutionTime", "QueryPlanningTime", "QueryQueueTime", "ServiceProcessingTime", "TotalExecutionTime"},
```
Comment (papagian) [inline pkg/tsdb/cloudwatch/metric_find_query.go:51]:
```suggestion
	"AWS/Athena":                  {"EngineExecutionTime", "ProcessedBytes", "QueryPlanningTime", "QueryQueueTime", "ServiceProcessingTime", "TotalExecutionTime"},
```

I think that `DataScannedInBytes` is no more around; at least it does appear in the docs.

**Review by papagian — CHANGES_REQUESTED** [review]
Generally LGTM; I've spotted an additional metric and an (obsolete?) one, WDYT?

File: pkg/tsdb/cloudwatch/metric_find_query.go (hunk 1 above)
Comment (kwarunek) [reply pkg/tsdb/cloudwatch/metric_find_query.go]: good catch

**Review by papagian — APPROVED** [review]
Thank you for your contribution!

**Comment by kylebrandt** [comment]
Removed cp label because it is in beta2, and see this in the v7.3.x branch already

**Pull request merged**